src/store/accounts.rs

```diff
@@ -95,12 +95,15 @@ impl Store {
         self.db
             .prepare(QUERY)
             .map_err(StoreError::QueryError)?
-            .query_map([], |row| row.get(0))
+            .query_map([], |row| {
+                let row: i64 = row.get(0)?;
+                Ok(row)
+            })
```

Comment:
I think these changes can be removed since `id` will be inferred to be `i64` because of the following map